tests/TestCase.php

```diff
@@ -247,8 +254,10 @@ public static function withSys($id, $data = [])
 
 class MockField extends \Contentful\Delivery\Resource\ContentType\Field
 {
-    public function __construct(array $data)
+    public function __construct($id, $name, $type, array $data = [])
```

Comment:
I've changed the constructor of `MockField` for ease of use, because ID, name and type are always set either way.